### PR TITLE
feat(StatusDot): add component

### PR DESCRIPTION
**Pull request by Mnau-mnau**

### CODE REVIEW

File: src/components/StatusDot/StatusDot.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  bad: '242, 46, 67',
+  neutral: '84, 129, 217',
+  inactive: '216, 216, 216',
+};
+
+const StatusDotComponent = Styled.div`
+  width: ${pxToRem(12)};
+  height: ${pxToRem(12)};
+  border-radius: ${getRadii('round')};
+  background-color: rgba(${({ color }) => StatusDotColors[color]});
```
Comment (ajkl2533) [inline src/components/StatusDot/StatusDot.tsx:18]:
```suggestion
  background-color: rgb(${({ color }) => StatusDotColors[color]});
```

File: src/components/StatusDot/StatusDot.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import PropTypes from 'prop-types';
+
+import { getRadii, pxToRem } from '../../utils';
+
+const StatusDotColors = {
+  good: '100, 244, 81',
+  bad: '242, 46, 67',
+  neutral: '84, 129, 217',
+  inactive: '216, 216, 216',
+};
```
Comment (ajkl2533) [inline src/components/StatusDot/StatusDot.tsx:12]:
```suggestion
} as const;
```

File: src/components/StatusDot/StatusDot.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  width: ${pxToRem(12)};
+  height: ${pxToRem(12)};
+  border-radius: ${getRadii('round')};
+  background-color: rgba(${({ color }) => StatusDotColors[color]});
+  border: 2px solid white;
+  box-shadow: 0 0 12px 0px rgba(${({ color }) =>
+    `${StatusDotColors[color]}, 0.5`});
+`;
+
+const StatusDot = ({ color }) => (
```
Comment (ajkl2533) [inline src/components/StatusDot/StatusDot.tsx:24]: Missing type annotation.

```suggestion
const StatusDot = ({ color }: {color: keyof typeof StatusDotColors}) => (
```

File: src/components/StatusDot/StatusDot.enums.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+export const StatusDotColors = {
+  good: 'good',
+  bad: 'bad',
+  neutral: 'neutral',
+  inactive: 'inactive',
+} as const;
```
Comment (ajkl2533) [inline src/components/StatusDot/StatusDot.enums.ts:6]: Let's kill this file. I think the .enums files doesn't serve the purpose in our reality.

File: src/components/StatusDot/StatusDot.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  <StatusDotComponent color={color} role="presentation" />
+);
+
+StatusDot.propTypes = {
+  color: PropTypes.string,
```
Comment (ajkl2533) [inline src/components/StatusDot/StatusDot.tsx:29]:
```suggestion
  color: PropTypes.oneOf(Object.keys(StatusDotColors))
```

File: src/components/StatusDot/StatusDot.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  height: ${pxToRem(12)};
+  border-radius: ${getRadii('round')};
+  background-color: rgb(${({ color }) => StatusDotColors[color]});
+  border: 2px solid white;
+  box-shadow: 0 0 12px 0px rgba(${({ color }) =>
+    `${StatusDotColors[color]}, 0.5`});
+`;
+
+const StatusDot = ({
+  color = 'inactive',
```
Comment (julesbarr) [inline src/components/StatusDot/StatusDot.tsx:25]: should we change the prop name to sentiment instead of color? since the values are not colors

Comment (ajkl2533) [reply src/components/StatusDot/StatusDot.tsx:25]: I personally wouldn't change the prop name, although I understand your point. `Color` is more predictable name than `sentiment` and values will be autocomplete by consumer IDE

**Review by ajkl2533 — APPROVED** [review]

**Pull request merged**